packages/visualizations/addon/models/table.js

```diff
@@ -124,6 +137,21 @@ function hasAllColumns(request, columns) {
     requestFields.every(field => columnFields.includes(field));
 }
 
+/**
+ * @function isSameColumn
+ * @param {Object} colA - column object
+ * @param {Object} colB - column object
+ * @returns {Boolean} - whether the columns are the same field
+ */
+function isSameColumn(colA, colB) {
+  if(colA.type === colB.type) {
+    return colA.type === 'dimension' ? 
+      colA.field.dimension == colB.field.dimension : 
+      canonicalizeMetric(colA.field) == canonicalizeMetric(colB.field);
```

Comment:
Also I feel like this logic is done in multiple places. Could we make use of displayName here instead? `colA.displayName === colB.displayName`